.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,51 @@
+name: Feature request
+description: Suggest an idea for this project
+labels: [enhancement]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for making NewPipe better by suggesting a feature. :hugs:
+
+        Your ideas are highly welcome! It's a tool for you, the users, after all.
+        
+  - type: checkboxes
+    id: checklist
+    attributes:
+      label: "Checklist"
+      options:
+        - label: "I made sure that there are *no existing issues*, neither [open](https://github.com/TeamNewPipe/NewPipe/issues) nor [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed), which I could contribute my information to."
+          required: true
+        - label: "This is a request for NewPipe itself. Requests for any new service need to be suggested at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues)."
+          required: true
+        - label: "I am going to take the time to to fill in all the required details. I know that the feature request will be dismissed otherwise."
```

Comment:
```suggestion
        - label: "I have taken the time to fill in all the required details. I understand that the feature request could be dismissed otherwise."
```